src/progly/query.cc

```diff
@@ -496,7 +529,52 @@ void worker()
                     }
 
                     case SFT_FLATBUF_UNION_ROW:
-                    case SFT_FLATBUF_UNION_COL:
+                    case SFT_FLATBUF_UNION_COL: {
+
+                        // extract ptr to meta data blob
+                        const char* blob_dataptr = reinterpret_cast<const char*>( meta.blob_data ) ;
+                        size_t blob_sz           = meta.blob_size ;
+                        //std::cout << "blob_sz = " << blob_sz << std::endl ;
```

Comment:
looks good.  i think you can remove this cout print and the other one below.

Reply:
done